android/keyframes/src/main/java/com/facebook/keyframes/KeyframesDrawable.java

```diff
@@ -317,6 +333,11 @@ public void setFrameProgress(float frameProgress) {
     }
   }
 
+  public void seekToProgress(float progress) {
+    stopAnimation();
+    onProgressUpdate(progress * 100);
```

Comment:
Yeah, the Android API definitely takes progress via frame values, there's a difference in the API on both platforms.  Probably it was hard to notice because the animation is close to 100 frames (88 frames), so playing 100 frames of it should just give us an extra 12 frames of the last frame of the animation, which is just a static logo anyway.